### ISSUE TITLE
-r GPU-accelerated

### ISSUE THREAD

**Issue by A-Kazak** [kimci86/bkcrack#75]
Greetings!

I would be great to have **-r** accelerated by GP GPUs.
Is there a plan to implement this?

Thank you.

**Comment by kimci86** [comment]
Hi,
Yes that would be a great improvement. I hope to learn OpenCL "soon" to speed up the computation.
This is related to #69.

**Comment by A-Kazak** [comment]
Got it!
IMHO CUDA will offer much more performance over OpenCL in this case.

**Comment by magnumripper** [comment]
I disagree. On the bottom line the only thing CUDA gives us is less portability. OpenCL is the way to go.
